sbncode/CAFMaker/FillTrue.cxx

```diff
@@ -109,6 +109,42 @@ namespace caf {
 
   }//FillSliceTruth
 
+
+ void FillMeVPrtlTruth(const evgen::ldm::MeVPrtlTruth &truth,
+                       caf::SRMeVPrtl &srtruth) {
+   // Fill stuff!!
+   srtruth.dcy.x = truth.decay_pos.X();
+   srtruth.dcy.y = truth.decay_pos.Y();
+   srtruth.dcy.z = truth.decay_pos.Z();
+   srtruth.dcyT  = truth.decay_pos.T();
+
+   srtruth.mom.x = truth.mevprtl_mom.X();
+   srtruth.mom.y = truth.mevprtl_mom.Y();
+   srtruth.mom.z = truth.mevprtl_mom.Z();
+   srtruth.E     = truth.mevprtl_mom.E();
+
+   srtruth.M = truth.mass;
+   srtruth.flux_weight = truth.flux_weight;
+   srtruth.ray_weight = truth.ray_weight;
+   srtruth.decay_weight = truth.decay_weight;
+   srtruth.C1 = truth.C1;
+   srtruth.C2 = truth.C2;
+   srtruth.C3 = truth.C3;
+   srtruth.C4 = truth.C4;
+   srtruth.C5 = truth.C5;
+
+   switch(truth.gen) {
+     case evgen::ldm::kDissonantHiggs:
```

Comment:
We need to change this following the changes in sbnanaobj.

Reply:
No actually the way I changed the code there's no change to sbncode. There is now an update to sbnana, which is in another PR.